packages/structure/src/interactive_cli/command_builder.ts

```diff
@@ -60,7 +60,7 @@ class CommandBuilder {
   }
 
   @lazy() get args() {
-    return this.opts.cmd.parsed._
+    return this.opts.cmd.parsed._ as string[]
```

Comment:
We needed this to be able to build and test but probably isn't required. TS was reporting this as a `(string | number)[]` and that was interfering with types elsewhere in this file.